src/backend/vulkan/VulkanBackend.cpp

```diff
@@ -103,6 +160,84 @@ namespace vulkan {
     void Device::TickImpl() {
     }
 
+    bool Device::CreateInstance(KnownGlobalVulkanExtensions* usedGlobals) {
+        std::vector<const char*> layersToRequest;
+        std::vector<const char*> extensionsToRequest;
+
+        #if defined(NXT_ENABLE_ASSERTS)
+            if (info.global.standardValidation) {
+                layersToRequest.push_back(kLayerNameLunargStandardValidation);
+                usedGlobals->standardValidation = true;
+            }
+            if (info.global.debugReport) {
+                extensionsToRequest.push_back(kExtensionNameExtDebugReport);
+                usedGlobals->debugReport = true;
+            }
+        #endif
+
+        VkApplicationInfo appInfo;
```

Comment:
If you `= {};` here, you could probably omit the `= nullptr` and `= 0`s below? If you want.

Reply:
I'd rather not omit such ` = 0`. Though the `pNext = nullptr` is annoying to write. Keeping as is if you don't mind.